README.md

```diff
@@ -85,23 +85,17 @@ Then download the template to the current directory:
 azd init -t Azure-Samples/azure-sql-db-session-recommender
```

Comment:
Add a comment that they only need to provide the -t parameter if they haven't already cloned it.

Reply:
Like we did here: https://github.com/jongio/chat-with-your-data-solution-accelerator/pull/2/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R131

Reply:
Updated.